tests/contrib/dbapi/test_dbapi.py

```diff
@@ -181,6 +182,50 @@ def method():
         assert span.get_metric('db.rowcount') == 123, 'Row count is set as a metric'
         assert span.get_metric('sql.rows') == 123, 'Row count is set as a tag (for legacy django cursor replacement)'
 
+    def test_cfg_service(self):
+        cursor = self.cursor
+        tracer = self.tracer
+        cursor.rowcount = 123
+        pin = Pin(None, app='my_app', tracer=tracer, tags={'pin1': 'value_pin1'})
+        cfg = AttrDict(service="cfg-service")
+        traced_cursor = TracedCursor(cursor, pin, cfg)
+
+        def method():
+            pass
+
+        traced_cursor._trace_method(method, 'my_name', 'my_resource', {'extra1': 'value_extra1'})
+        span = tracer.writer.pop()[0]  # type: Span
+        assert span.service == 'cfg-service'
+
+    def test_default_service(self):
+        cursor = self.cursor
+        tracer = self.tracer
+        cursor.rowcount = 123
+        pin = Pin(None, app='my_app', tracer=tracer, tags={'pin1': 'value_pin1'})
+        traced_cursor = TracedCursor(cursor, pin, None)
+
+        def method():
+            pass
+
+        traced_cursor._trace_method(method, 'my_name', 'my_resource', {'extra1': 'value_extra1'})
+        span = tracer.writer.pop()[0]  # type: Span
+        assert span.service == "db"
+
+    def test_default_service_cfg(self):
+        cursor = self.cursor
+        tracer = self.tracer
+        cursor.rowcount = 123
+        pin = Pin(None, app='my_app', tracer=tracer, tags={'pin1': 'value_pin1'})
```

Comment:
Should we include tests for when the `Pin.service` is set as is `config.<dbapi-integration>.{_default_service,service}`?

Reply:
done for dbapi, not sure if it's worth adding for all of the integrations since they're using the helper